.github/workflows/macos.yml

```diff
@@ -17,7 +17,7 @@ env:
   CMAKE_BUILD_PARALLEL_LEVEL: 3
   CTEST_PARALLEL_LEVEL: 3
   CTEST_OUTPUT_ON_FAILURE: 1
-  CXXFLAGS: "-Wall -Wextra -Wpedantic"
+  CXXFLAGS: "-Wall -Wextra -Wpedantic -march=broadwell"
```

Comment:
Not only does this not work in the CI, but will probably also break compatibility with M1. Better leave it out completely rather than trying to find a workaround.

Reply:
I agree. Another question: shall we set `WITH_LTO` to `ON` as default in the main `CMakeLists.txt`?

Reply:
In my experience `WITH_LTO` just inflates the build times by something like an order of magnitude without any large gains in performance.